### DISCUSSION TITLE
About supporting Minecraft 1.19.3 and 1.19.4

### DISCUSSION THREAD

**Discussion by Shiho320** [vectorwing/FarmersDelight#724] [General]
Is there any plan to update the mod to Minecraft 1.19.3 and 1.19.4? I have been always waiting for it.

**Comment by vectorwing** [comment]
We will wait for 1.20 to release, rather than port to .3 or .4 currently. 👍

**Reply by ghost** [reply]
Will the update be soon?

**Reply by vectorwing** [reply]
It is ongoing, but there's no estimated date for release. Please don't ask for ETA's. ^^
